src/drawbridge.rs

```diff
@@ -0,0 +1,114 @@
+// SPDX-License-Identifier: Apache-2.0
+
+use std::fs::read_to_string;
+use std::str::FromStr;
+
+use anyhow::Context;
+use drawbridge_client::types::{RepositoryContext, TagContext, UserContext};
+use drawbridge_client::Client;
+use rustls::{Certificate, RootCertStore};
+
+const DEFAULT_HOST: &str = "example.com";
```

Comment:
What is this for and why would we want that in production code?

Reply:
This is just the default drawbridge, which we certainly want to support, but I don't the actual address of our production drawbridge instance.